core/server/data/schema/schema.js

```diff
@@ -205,8 +205,18 @@ module.exports = {
             defaultTo: 'public',
             validations: {isIn: [['public', 'internal']]}
         },
+        og_image: {type: 'string', maxlength: 2000, nullable: true},
+        og_title: {type: 'string', maxlength: 300, nullable: true},
+        og_description: {type: 'string', maxlength: 500, nullable: true},
+        twitter_image: {type: 'string', maxlength: 2000, nullable: true},
+        twitter_title: {type: 'string', maxlength: 300, nullable: true},
+        twitter_description: {type: 'string', maxlength: 500, nullable: true},
         meta_title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}},
         meta_description: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 500}}},
+        codeinjection_head: {type: 'text', maxlength: 65535, nullable: true},
+        codeinjection_foot: {type: 'text', maxlength: 65535, nullable: true},
+        canonical_url: {type: 'text', maxlength: 2000, nullable: true},
+        color: {type: 'string', maxlength: 50, nullable: true},
```

Comment:
Not sure about the naming here. [Settings uses](https://github.com/TryGhost/Ghost/blob/8c8af3b4ac95d4a4da05488295c812bae768a1d4/core/server/data/schema/default-settings.json#L77-L81) `accent_color` as a name for this property and the spec mentions an `accent_color` helper in the future. The only place I see the rename is in future Admin UI but that doesn't imply change in the db naming :thinking:

Reply:
+1 on using `accent_color` to be consistent with both spec and settings usage :)